packages/core/src/utils/useFloatingUI/useFloatingUI.tsx

```diff
@@ -71,6 +77,20 @@ const DefaultFloatingComponent = forwardRef<
 
   const ChosenSaltProvider = themeNext ? SaltProviderNext : SaltProvider;
 
+  useEffect(() => {
+    if (!disableScroll) {
+      return;
+    }
+    if (open) {
+      document.documentElement.style.overflow = "hidden";
```

Comment:
we should probably use `targetWindow`

Reply:
Hi - I've updated it to use `targetWindow`.